blackbox/docs/getting_started/clients/java.txt

```diff
@@ -0,0 +1,103 @@
+=============================
+Connecting to Crate with Java
+=============================
+
+Java Client
+-----------
+
+Deprecation Notice
+~~~~~~~~~~~~~~~~~~
+
+The Java client is deprecated in versions of Crate 0.57.0 and will be
+removed by 0.58.0 and later. Instead the recommended way to access Crate
```

Comment:
The Java client was deprecated in versions of Crate **0.57.0** and is remove in **0.58** ..